frontend/src/assets/i18n/en.json

```diff
@@ -562,6 +562,11 @@
             },
             "comparison": {
                 "legend-desc": "The image shows the relative difference in total cumulative impact between the base and what-if scenario."
+            },
+            "overflow": {
+                "title": "Cumulative impact exceeds maximum value",
+                "detail": "Note that the part of the cumulative result that has exceeded the maximum value per pixel is not reflected in the report. Hence, the result for the affected bands is lower than the value would be if the entire parameterized change was factored in, without limiting each pixel value.",
+                "list-description": "The following bands would exceed maximum value :"
```

Comment:
The reason for using the word "would" is intentional, what's happening is that when a change to some band value _would_ bring the pixel's value over max (100), in reality it isn't - since all pixel values are 'capped' at the set max.
This 'capping' makes the actual result lower than what might be expected from an intensity change, the purpose of this development is to draw the user's attention to that.

Reply:
Although, I see that the wording ("would") is inconsistent with what I've put for the Swedish and French translations. It's harder to convey this in Swedish, especially since we need a succinct phrase. I may choose to go with your suggestion anyway and keep it consistent. It should be clear from the more detailed explanation (labeled `"detail"`).

There is also a ongoing discussion about the wording of the information blurb on the internal work tickets' message board.

Reply:
I think I'll be settling for `"The following changes exceed maximum value:"` which may be seen as more exact. How about it?

Reply:
I understand. I know the context is there, the statement simply feels incomplete. I was left wondering if I missed something because the word 'would' implied a clause that I was worried I missed. 

**Based on context and the informed user it can probably be left as is**. 
If it comes up at a later date, one possibly could write:
"Without the pixel value limits, the following bands would exceed maximum :"

Reply:
The problem with a more verbose sentence in this case, is that it's a little long for its placement in the layout.

Reply:
I see we responded at the same time earlier. I think your shorter phrase > "The following changes exceed maximum value:" is prefect. :)